docs/UserGuide.md

```diff
@@ -115,32 +116,51 @@ Displays the output of `View` command, which expands each ingredient or recipe c
   e.g. `add ingredient -n <name> -q <quantity>[<units>]` can be used as:
      1. `add ingredient -n chicken -q 2` or as
      2. `add ingredient -n chicken -q 2kg`
+    
+- Items in parentheses `()` should come together.
+
+    e.g. `edit ingredient <index> (-<field flag> <new data>)...` can be used as:
+     1. `edit ingredient 3 -q 3` or as 
+     2. `edit ingredient 3 -q 3 -e 11-03-2022`
 
 - Items with `…` after them can be used multiple times.
 
   e.g. `find ingredient <keyword>...`, can be used as:<br />`find ingredient Strawberry Milk Cheese Tomato`
 
-## 4.1 General Commands
+## 4.2 Command Flags
+
+ Flag | Usage | Description | Compulsory? | Remarks 
+---- | ------- | ----------- | ------- | ----
+`-n` | `-n <name>` | Name of the ingredient or recipe | Yes | Names can only contain alphanumeric characters or spaces.
+`-q` | `-q <quantity> [unit]` | Quantity of the ingredient | Yes | Must be a number. Can be followed by a unit of measurement (`g`, `kg`, `mg`, `l`, `ml`, `kl`). Quantities will be converted to grams or litres, and numbers rounded off to 3 decimal places.
+`-i` | `-i <name> <quantity>...` | Ingredients used in the recipe | At least one specified | Must be a name followed by a space and a quantity.
+`-e` | `-e <expiry date>` | Expiry date of ingredients | Yes | Must be in the form DD-MM-YYYY.
+`-s` | `-s [step]...` | Step used in the recipe | No | Can contain any characters or spaces. 
+`-d` | `-d [description]` | Description of the recipe or ingredient | No | Can contain any characters or spaces.
+`-t` | `-t [tag]...` | Tag for the ingredient | No | Can contain alphanumeric characters or spaces.
```

Comment:
how come the notation went from `<name>` to `[step]`. If you want to show any parts as optional, it should not be here. It should be specific to the commands themselves and not at the command flag explanations.

Reply:
Should mention expired and expiring and reserved tags for fridgy. They will not allowed to be used by user. Either in this table or in the add ingredient feature section